### PR TITLE
Provide concept guide for the Kanto Suite Connector

### PR DESCRIPTION
**Pull request by antoniyatrifonova**
[#11] Provide concept guide for the Kanto Suite Connector

- Provide information about what is a Suite Connector and how it works
- Provide an illustration of this concept

Signed-off-by: Antonia Trifonova Antonia.Trifonova@bosch.io

### CODE REVIEW

File: web/site/content/docs/concepts/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Concepts"
+type: docs
```
Comment (e-grigorov) [inline web/site/content/docs/concepts/_index.md:3]: Description with a single sentence is required here as it's for the getting started guide.

File: web/site/content/docs/concepts/suite-connector.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Suite connector"
+type: docs
+description: >
+  Enables the cloud connectivity and local communication.
```
Comment (e-grigorov) [inline web/site/content/docs/concepts/suite-connector.md:5]: - Imperative verb should be used here as it's for the getting started guide i.e. it should be: Enable...
- Suite connector doesn't enable the local communication. It's part of it. I would suggest to remove it.

File: web/site/content/docs/concepts/suite-connector.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+weight: 1
+---
+
+Suite connector enables the remote connectivity to an IoT cloud ecosystem of choice, powered by Eclipse Hono™ (e.g. [Eclipse Cloud2Edge](https://www.eclipse.org/packages/packages/cloud2edge) and Bosch IoT Suite). It provides the following use cases:
```
Comment (e-grigorov) [inline web/site/content/docs/concepts/suite-connector.md:9]: The link can be opened in a new tab. In this way, the reader will keep the focus on the concept. refn shortcode can be used. Getting started guide contains a few examples.

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: web/site/content/docs/concepts/suite-connector.md [hunk 4]
```diff
@@ -2,11 +2,11 @@
 title: "Suite connector"
 type: docs
 description: >
-  Enables the cloud connectivity and local communication.
+  Empower the edge device with a remote connectivity.
 weight: 1
 ---
 
-Suite connector enables the remote connectivity to an IoT cloud ecosystem of choice, powered by Eclipse Hono™ (e.g. [Eclipse Cloud2Edge](https://www.eclipse.org/packages/packages/cloud2edge) and Bosch IoT Suite). It provides the following use cases:
+Suite connector enables the remote connectivity to an IoT cloud ecosystem of choice, powered by Eclipse Hono™ (e.g. <a href="https://www.eclipse.org/packages/packages/cloud2edge" target="_blank">Eclipse Cloud2Edge</a>. and Bosch IoT Suite). It provides the following use cases:
```
Comment (e-grigorov) [inline web/site/content/docs/concepts/suite-connector.md:9]: My point was about refn shortcode. Here are some examples:
https://raw.githubusercontent.com/eclipse-kanto/kanto/main/web/site/content/docs/getting-started/hono.md
i.e. {{% refn "https://www.eclipse.org/packages/packages/cloud2edge" %}}Eclipse Cloud2Edge{{% /refn %}} should be used instead.

**Review by e-grigorov — CHANGES_REQUESTED** [review]

**Review by e-grigorov — APPROVED** [review]

**Review by konstantina-gramatova — APPROVED** [review]
LGTM

**Pull request merged**